tvu_net_cron/views/.ipynb_checkpoints/cancel_expired_quotation_views-checkpoint.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <data noupdate="1">
+        <record model="ir.cron" id="cancel_expired_quotation_view_form">
+            <field name="name">cancel_expired_quotation</field>
```

Comment:
Names of crons should be normal descriptions of what the cron does, so "Cancel Expired Quotations"